### DISCUSSION TITLE
Lowering minimum grid snap spacing to 1 unit

### DISCUSSION THREAD

**Discussion by KSRVIIIXVIIIII** [ppy/osu#29907] [Ideas] [answered]
The issue I'm having is that the notes handplaced in lazer beatmap editor are most likely to be with decimal points in their coordinates.
I mainly play in osu stable so, if I were to import the map I have done in lazer to stable, the coordinates of the notes would be truncated into integers and the notes would be moved. I found it quite annoying to see the placements of notes in lazer differ from that in stable.
However, the minimum grid snap spacing is at 4 units, and so I couldn't make sub-unit movements to the notes while keeping the position identical.

So, I would like to ask for lowering the minimum grid snap spacing to 1 unit, so that sub-unit movements for the notes can be done and no decimal coordinates for notes as well. Thanks

**Comment by peppy** [comment, marked as answer]
Probably fine to do this as long as the UX isn't destroyed by adding such a low value.
